internal/cli/cmd/cluster/buildx.go

```diff
@@ -318,11 +410,19 @@ func banner(ctx context.Context, name string, use bool, native []api.BuildPlatfo
 		fmt.Fprintf(w, "\n  docker buildx use %s\n", name)
 	}
 
-	fmt.Fprintf(w, "\nStart a new terminal, and start building:\n")
-	fmt.Fprintf(w, "\n  docker buildx build ...\n")
+	if !background {
+		fmt.Fprintf(w, "\nStart a new terminal, and start building:\n")
+		fmt.Fprintf(w, "\n  docker buildx build ...\n")
 
-	fmt.Fprintln(w)
-	fmt.Fprintln(w, style.Comment.Apply("Exiting will remove the configuration."))
+		fmt.Fprintln(w)
+		fmt.Fprintln(w, style.Comment.Apply("Exiting will remove the configuration."))
+	} else {
+		fmt.Fprintf(w, "\nStart building:\n")
+		fmt.Fprintf(w, "\n  docker buildx build ...\n")
+
+		fmt.Fprintf(w, "\nThe proxy is running in background. You can stop it and unconfigure the remote builders with:\n")
```

Comment:
"proxy" is an implementation detail, I'd avoid making references to it. I suggest saying:

"Your remote builder context is running in the background. You can always clean up previous runs with: nsc docker buildx cleanup".